sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -141,6 +141,18 @@ def wait_for_hand_params(self, timeout_in_secs):
             self._hand_h = True
             self._hand_h_parameters = rospy.get_param("/fh_hand")
 
+    def is_type_hand_e(self):
+        return self._hand_e
+
+    def get_hand_e_parameters(self):
+        return self._hand_parameters
```

Comment:
Is this variable only related to hand e?

Reply:
The class had only hand_h and hand_e related params, so I didnt extend with lite etc.
hand_parameters is for hand_e, hand_e_parameters is for hand_h (you can check in init of the class)

Reply:
If so, please make this variable more meaningful

Reply:
> The class had only hand_h and hand_e related params, so I didnt extend with lite etc. hand_parameters is for hand_e, hand_e_parameters is for hand_h (you can check in init of the class)

This does not make sense does it?

Reply:
sorry, edited (hand_e_parameters -> hand_h_parameters)

Reply:
It seems the issue is that code elsewhere is still trying to access this parameter directly. Once all of it is changed to use the new getter this parameter could be changed safely